READEME.md

```diff
@@ -0,0 +1,15 @@
+# Moving Square
+
+App made only in JavaScript for training skills.
+
+## Start App
+
+-App is ready to use and starting automatically.
```

Comment:
Add space after dash, it will be interpreted in MD as unordered list https://www.markdownguide.org/cheat-sheet/

Reply:
@Miciula and still not fixed